subiquity/common/filesystem/tests/test_manipulator.py

```diff
@@ -503,3 +503,28 @@ def test_add_boot_PREP_full_resizes_larger(self):
         disk = make_disk(manipulator.model, preserve=True, size=2002*MiB)
         self._test_add_boot_full_resizes_larger(
             manipulator, disk, sizes.PREP_GRUB_SIZE_BYTES)
+
+
+class TestReformat(unittest.TestCase):
+    def setUp(self):
+        self.manipulator = make_manipulator()
+
+    def test_reformat_default(self):
+        disk = make_disk(self.manipulator.model, ptable=None)
+        self.manipulator.reformat(disk)
+        self.assertEqual(None, disk.ptable)
+
+    def test_reformat_keep_current(self):
+        disk = make_disk(self.manipulator.model, ptable='msdos')
+        self.manipulator.reformat(disk)
+        self.assertEqual('msdos', disk.ptable)
```

Comment:
I realise this looks like the current behaviour but is this actually what we want? In my mind we always reformat to GPT but maybe that's not what happens...

Reply:
Yeah that makes sense.